scripts/funcotator/data_sources/createLiftoverForB37ToHg38.sh

```diff
@@ -0,0 +1,567 @@
+#!/usr/bin/env bash
+
+################################################################################
+#
+# WARNING: THIS SCRIPT IS UNSUPPORTED!
```

Comment:
Can you move these scripts into a directory structure that contains the word "unsupported"?

Reply:
Yeah.  I think I had a different understanding of what "Unsupported" means.

Reply:
I have added a file named `ALL_SCRIPTS_ARE_UNSUPPORTED.txt` to the top folder here.